src/Essentials/src/Geolocation/Geolocation.shared.cs

```diff
@@ -27,6 +29,40 @@ public interface IGeolocation
 		/// <remarks>The location permissions will be requested at runtime if needed. You might still need to declare something in your app manifest.</remarks>
 		/// <returns>A <see cref="Location"/> object containing current location information or <see langword="null"/> if no location could be determined.</returns>
 		Task<Location?> GetLocationAsync(GeolocationRequest request, CancellationToken cancelToken);
+
+		/// <summary>
+		/// Indicates if currently listening to location updates while the app is in foreground.
+		/// </summary>
+		bool IsListeningForeground { get; }
+
+		/// <summary>
+		/// Occurs while listening to location updates.
+		/// </summary>
+		event EventHandler<LocationEventArgs>? LocationChanged;
```

Comment:
Not sure if this should be LocationChanged or LocationUpdated...

Reply:
It's up to you, either sounds good.

Reply:
@mattleibow If you have no preference I would let the event as is and just rename the event args class, as requested above.

Reply:
Sounds good.